### PR TITLE
bitnami/dataplatform-bp2 kafka-spark-elasticsearch

### PR DESCRIPTION
**Pull request by gkulkarni32**
**Description of the change**

New Orchestration helm chart for Data Platform comprising of Kafka-Spark-Elasticsearch along with optional Wavefront Observability framework.

**Benefits**

Users will be able to deploy a data platform comprising of kafka, Spark, Elasticsearch with Kibana and Logstash along with Wavefront Observability framework.

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/dataplatform-bp2/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    version: 6.x.x
+  - name: common
+    repository: https://charts.bitnami.com/bitnami
+    tags:
+      - bitnami-common
+    version: 1.x.x
+description: OCTO Data platform Kafka-Spark-Elasticsearch Helm Chart
+engine: gotpl
+home: https://github.com/bitnami/charts/tree/master/bitnami/dataplatform-bp2
+icon: https://bitnami.com/assets/stacks/dataplatform-bp1/img/dataplatform-bp1-stack-220x234.png
```
Comment (andresbono) [inline bitnami/dataplatform-bp2/Chart.yaml:34]: The link to the icon might need an update, leaving this comment for future review.

File: bitnami/dataplatform-bp2/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    version: 5.x.x
+  - condition: elasticsearch.enabled
+    name: elasticsearch
+    repository: https://charts.bitnami.com/bitnami
+    version: 14.x.x
+  - condition: logstash.enabled
+    name: logstash
+    repository: https://charts.bitnami.com/bitnami
+    version: 3.x.x    
+  - condition: zookeeper.enabled
```
Comment (andresbono) [inline bitnami/dataplatform-bp2/Chart.yaml:22]: ZooKeeper is only used in Kafka, right? Would it make more sense to use the ZooKeeper as subchart of Kafka by setting `kafka.zookeeper.enabled` to `true` (the default value)?

**Review by andresbono — COMMENTED** [review]
Thanks for submitting this PR! I have just added a question to set the basis for the PR review.

File: bitnami/dataplatform-bp2/Chart.yaml (hunk 2 above)
Comment (gkulkarni32) [reply bitnami/dataplatform-bp2/Chart.yaml:22]: Yes, I will make this change.

**Comment by marcosbc** [comment]
Hi, I'm seeing some recent changes not related to the code review. Is this ready to be re-checked again?

**Comment by gkulkarni32** [comment]
Hi @marcosbc 
I had a word with Miguel last week, had updated a values according to our discussion. It's ready to be reviewed. Please go ahead. 

Thanks
Geeta

**Pull request closed without merging**